### DISCUSSION TITLE
Changie mentioned

### DISCUSSION THREAD

**Discussion by sheldonhull** [miniscruff/changie#484] [Show and tell]
https://dagger.io/blog/how-dagger-releases

Been promoting where I can. Using it for "Changelog driven releases" with mage and Goreleaser and it's working great. Cheers! 

Also love the new block editing. 🙏

**Comment by miniscruff** [comment]
> "Changelog driven releases"

This is a great term for what I have been doing and documenting as well, thanks for the share.

**Reply by sheldonhull** [reply]
They just adopted :tada: 

https://github.com/dagger/dagger/pull/5408

**Reply by sheldonhull** [reply]
https://marketplace.visualstudio.com/items?itemName=Continue.continue

Recommended and they ended up adopting after talking about the benefits. Thought you'd like to know 👍

**Reply by miniscruff** [reply]
> https://marketplace.visualstudio.com/items?itemName=Continue.continue
> 
> Recommended and they ended up adopting after talking about the benefits. Thought you'd like to know 👍

LETS GO.
